### ISSUE TITLE
"Restore Tab" not working

### ISSUE THREAD

**Issue by Xerono** [Robbendebiene/Gesturefy#612]
When trying to restore a tab via gesture, nothing happens. Reopening it via Firefox works without problems.

- Operating system: Windows 10
- Firefox version: 92.0 (64 Bit)
- Gesturefy version: 3.2.1

Log: [Screenshot for better readbility](https://i.imgur.com/nQx7bjt.png)

`Uncaught (in promise) Error: An unexpected error occurred undefined
    RestoreTab moz-extension://5c03fecc-a3ce-4b5b-8720-47696cb91a82/core/commands.mjs:197
    AsyncFunctionThrow self-hosted:696
    (Async: async)
    execute moz-extension://5c03fecc-a3ce-4b5b-8720-47696cb91a82/core/models/command.mjs:67
    handleMouseGestureCommandExecution moz-extension://5c03fecc-a3ce-4b5b-8720-47696cb91a82/core/background.mjs:109
    <anonym> moz-extension://5c03fecc-a3ce-4b5b-8720-47696cb91a82/core/background.mjs:61
    apply self-hosted:2736
    raw resource://gre/modules/ExtensionCommon.jsm:2437
    wrapResponse resource://gre/modules/ExtensionChild.jsm:221
    responses resource://gre/modules/ExtensionChild.jsm:190
    map self-hosted:221
    emit resource://gre/modules/ExtensionChild.jsm:190
    recvRuntimeMessage resource://gre/modules/ExtensionChild.jsm:359
    recvRuntimeMessage self-hosted:1175
    _recv resource://gre/modules/ConduitsChild.jsm:78
    receiveMessage resource://gre/modules/ConduitsChild.jsm:184
Uncaught (in promise) Error: Could not establish connection. Receiving end does not exist.
    sendContentTabId moz-extension://3f15d020-6244-4d13-83e4-beee560c83c1/reloadmatic.js:463
reloadmatic.js:463:25
Uncaught (in promise) Error: Could not establish connection. Receiving end does not exist.
    <anonymous> moz-extension://32d791f4-44f4-4589-befc-59d274599bf2/background.js:374
background.js:374:6512`

**Comment by Robbendebiene** [comment]
Thanks for the error message. However looking at the stack trace this looks like an error from some Firefox API internals.
Does the error still occur after a browser restart?

**Comment by Xerono** [comment]
Yes, it's happening for a few days now. If no one else has this problem, I assume something with my setup is wrong, and will investigate further.

**Comment by Robbendebiene** [comment]
Thanks for the update!
